packages/frontend-2/lib/viewer/composables/setup/postSetup.ts

```diff
@@ -328,7 +333,7 @@ function useViewerCameraIntegration() {
       // Only now set projection, we can't do it too early
       orthoProjectionUpdate(isOrthoProjection.value)
     } else {
-      loadCameraDataFromViewer()
+      if (!diff.diffString.value) loadCameraDataFromViewer() // NOTE: a bit of a hack
```

Comment:
for some reason, threads with diffs, when opened up, fail to set the correct view IF there's another model loaded in the scene besides the ones being diffed..

Reply:
this should be a comment in the code imo